data/etc/plinth/modules-enabled/diaspora

```diff
@@ -1,2 +0,0 @@
-plinth.modules.diaspora
```

Comment:
Would it make sense to leave this file in place, but comment out the line? That's what we do for restore and shaarli:
https://sources.debian.net/src/plinth/0.14.0%2Bds-1/debian/patches/0003-disable-missing-packages.patch/